test.js

```diff
@@ -55,6 +67,7 @@ test('allows deactivating capture (opts.dsn to be false)', async t => {
   await server.inject({
     method: 'GET',
     url: '/',
+    payload: t.title,
```

Comment:
did you add all those `payload`s for typescript reasons? Otherwise there is no obvious reason to me, reading this diff.